h2o-genmodel/src/main/java/hex/genmodel/easy/DWImageConverter.java

```diff
@@ -0,0 +1,68 @@
+package hex.genmodel.easy;
+
+import hex.genmodel.GenModel;
+import hex.genmodel.algos.deepwater.DeepwaterMojoModel;
+import hex.genmodel.easy.exception.PredictException;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Map;
+
+public class DWImageConverter extends RowToRawDataConverter {
+
+  private final DeepwaterMojoModel _dwm;
+  
+  DWImageConverter(DeepwaterMojoModel m, Map<String, Integer> modelColumnNameToIndexMap, Map<Integer, CategoricalEncoder> domainMap,
+                   EasyPredictModelWrapper.ErrorConsumer errorConsumer, EasyPredictModelWrapper.Config config) {
+    super(m, modelColumnNameToIndexMap, domainMap, errorConsumer, config);
+    _dwm = m;
+  }
+
+  @Override
+  protected boolean convertValue(String columnName, Object o, CategoricalEncoder catEncoder, int targetIndex, double[] rawData) throws PredictException {
+    BufferedImage img = null;
+
+    if (o instanceof String) {
+      String s = ((String) o).trim();
+      // Url to an image given
+      boolean isURL = s.matches("^(https?|ftp|file)://[-a-zA-Z0-9+&@#/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|]");
```

Comment:
Pre compile the pattern for better performance? JVM should instantiate lazily anyway.